src/common/Axis/LinearAxis/LinearAxis.story.tsx

```diff
@@ -2,73 +2,16 @@ import React from 'react';
 import { largeSignalChartData, categoryData } from '../../../../demo';
 import { ScatterPlot } from '../../../ScatterPlot';
 import { BarChart } from '../../../BarChart';
-import {
-  LinearYAxis,
-  LinearYAxisTickSeries,
-  LinearYAxisTickLine,
-  LinearYAxisTickLabel
-} from './LinearYAxis';
-import {
-  LinearXAxis,
-  LinearXAxisTickSeries,
-  LinearXAxisTickLine,
-  LinearXAxisTickLabel
-} from './LinearXAxis';
+import { LinearYAxis, LinearYAxisTickSeries, LinearYAxisTickLine, LinearYAxisTickLabel } from './LinearYAxis';
+import { LinearXAxis, LinearXAxisTickSeries, LinearXAxisTickLine, LinearXAxisTickLabel } from './LinearXAxis';
 
 export default {
   title: 'Utils/Axis/Linear'
 };
 
 export const CenterAxes = () => {
   const data = [...largeSignalChartData].splice(0, 50);
-  return (
-    <ScatterPlot
-      height={400}
-      width={400}
-      data={data}
-      gridlines={null}
-      yAxis={
-        <LinearYAxis
-          type="value"
-          position="center"
-          tickSeries={
-            <LinearYAxisTickSeries
-              line={<LinearYAxisTickLine position="center" />}
-              label={<LinearYAxisTickLabel padding={3} />}
-            />
-          }
-        />
-      }
-      xAxis={
-        <LinearXAxis
-          type="category"
-          position="center"
-          tickSeries={
-            <LinearXAxisTickSeries
-              line={<LinearXAxisTickLine position="center" />}
-              label={<LinearXAxisTickLabel padding={3} />}
-            />
-          }
-        />
-      }
-    />
-  );
+  return <ScatterPlot height={400} width={400} data={data} gridlines={null} yAxis={<LinearYAxis type="value" position="center" tickSeries={<LinearYAxisTickSeries line={<LinearYAxisTickLine position="center" />} label={<LinearYAxisTickLabel padding={3} />} />} />} xAxis={<LinearXAxis type="category" position="center" tickSeries={<LinearXAxisTickSeries line={<LinearXAxisTickLine position="center" />} label={<LinearXAxisTickLabel padding={3} />} />} />} />;
```

Comment:
Curious why you changed this?

Reply:
I think this is just a formatting issue - all props are the same, and the storybook examples run as expected, but it's auto-formatting everything on 1 line for me. Probably need to change some Prettier setting